README.md

```diff
@@ -536,9 +529,39 @@ This allows multiple configs to be generated, for example for different
 applications, and then using copies of a bare OS image of a particular version
 to prepare images for each of those apps.
 
-... Instructions on `config generate` and `os config` ...
+You can generate independent configuration files using the
+`balena config generate` command. This works in a similar way to the
+`balena os configure` command should only an OS version along with an
+application name or device UUID be specified, in that it will interactively ask
+for more details:
+```
+$ balena config generate --version 2.38.0+rev1 --application cliApp
+? Network Connection (Use arrow keys)
+❯ ethernet
+  wifi
+```
+
+However, in most situations where you wish to programmatically define a set of
+configurations, you can use other switches to do so. We'll use the `--output`
+switch to write a JSON configuration, and a few of the network configuration
+switches to ensure it connects to a Wifi Access Point on startup. In the
+following command, ensure you replace the values for `--wifiKey` and
+`--wifiSsid` with values for your local network's Access Point:
+```
+$ balena config generate --version 2.38.0+rev1 --application cliApp --appUpdatePollInterval 10 --network wifi --wifiSsid MyNetworkSSID --wifiKey myw1f1n3tw0rk -o wifi-config.json
+```
+This will generate a new JSON file in the current directory. We'll now use this
+to write to the downloaded image. You can either overwrite the current config
+if you've already configured it interactively in the step above, or use the
+freshly downloaded image:
```

Comment:
On the point of a "freshly downloaded image", perhaps worth noting that it the image already contained a WiFi connection profile, chances are it is at `system-connections/resin-wifi-01`, and running `balena os configure` will create an additional connection profile, possibly for the same WiFi network, at `system-connections/resin-wifi`. :-(

Reply:
That's a really good point, I'll edit this. :)